core/src/main/java/org/infinispan/container/EvictionSetup.java

```diff
@@ -0,0 +1,53 @@
+package org.infinispan.container;
+
+import java.util.Collections;
+
+import org.infinispan.commons.util.EvictionListener;
+import org.infinispan.container.entries.ImmortalCacheEntry;
+import org.infinispan.container.entries.InternalCacheEntry;
+import org.infinispan.util.concurrent.WithinThreadExecutor;
+
+import com.github.benmanes.caffeine.cache.CacheWriter;
+import com.github.benmanes.caffeine.cache.Caffeine;
+import com.github.benmanes.caffeine.cache.RemovalCause;
+
+/**
+ * @author wburns
+ * @since 9.3
+ */
+public class EvictionSetup {
```

Comment:
I might be missing something, but according to my IDE this class and it's methods are never used.

Reply:
Yeah this was refactored away when I added the Abstract class. Let me remove this.